src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: Query time parsing in logs
+tags:
+  - Logs
+  - Log management
+  - UI and data
+metaDescription: How New Relic uses parsing and how to send customized log data.
+freshnessValidatedDate: 2024-05-15 
+---
+
+
+
+
+import logsLogsTableEntryPoint from 'images/logs_screenshot-full_logs-table-entry-point.webp'
+
+import logsLogsDetailEntryPoint from 'images/logs_screenshot-full_logs-detail-entry-point.webp'
+
+import logsOpenQueryTimeParsingEditor from 'images/logs_screenshot-full_open-query-time-parsing-editor.webp'
+
+import logsExtractValuesinParserRule from 'images/logs_screenshot-crop_extract-values-in-parser-rule.webp'
+
+import logsEditorShowsParsedDataPreview from 'images/logs_screenshot-crop_editor-shows-parsed-data-preview.webp'
+
+import logsNrqlBehindQueryRule from 'images/logs_screenshot-crop_nrql-behind-query-rule.webp'
+
+import logsDeleteQueryTimeRule from 'images/logs_screenshot-crop_delete-query-time-rule.webp'
+
+import logsViewResultsinLogDetailsView from 'images/logs_screenshot-full_view-results-in-log-details-view.webp'
+
+import logsLogTableResults from 'images/logs_screenshot-full_log-table-results.webp'
+
+import logsRuleManager from 'images/logs_screenshot-crop_rule-manager.webp'
+
+import logsQueryTimeSearchBar from 'images/logs_screenshot-full_query-time-search-bar.webp'
+
+import logsAlertQueryBuilder from 'images/logs_screenshot-crop_alert-query-builder.webp'
+
+import logsQueryBehindParsingRule from 'images/logs_screenshot-crop_query-behind-parsing-rule.webp'
+
+import logsParsingAttributesView from 'images/logs_screenshot-crop_parsing-attributes-view.webp'
+
+
+
+
+
+Query time parsing lets you parse your logs directly in the UI without needing to write complex regular expressions or grok patterns. Results are shown instantly since parsing is performed at query time. You can use query time parsing to temporarily extract values from your logs and quickly perform a query on these variables.
+
+## How query time parsing differs from ingest parsing [#differences]
+
+While both types of parsing make it easier for you to query logs, they have some significant differences:
+
+**Ingest parsing:** Parsing during [log ingestion](/docs/logs/ui-data/parsing/) is where you create parsing rules using grok and/or regex. As log records are ingested at New Relic, your parsing rules are applied to create permanent attributes that are stored with your log data in NRDB. These attributes make it easier for you to query log data. 
```

Comment:
Can you indent these or bullet them? Right now, since they're left-justified, they don't feel like you're listing two significant differences. They should stand out more visually

Reply:
@homelessbirds good suggestion! I will indent these!